ground/src/main/res/values/colors.xml

```diff
@@ -37,4 +37,14 @@
 
   <!-- Data Submission Confirmation -->
   <color name="colorDataSubmissionConfirmationGreen">#216C2E</color>
+
+  <!-- Task UI Components -->
+  <color name="taskBottomControlsBackground">#FCFDF7</color>
+  <color name="taskBtnDarkGreen">#47854B</color>
+  <color name="taskBtnTextDarkGreen">#FFFFFF</color>
+  <color name="taskBtnLightGreen">#D5E8D0</color>
+  <color name="taskBtnTextLightGreen">#000000</color>
+  <color name="taskBtnTransparentStrokeColor">#747775</color>
+  <color name="taskBtnTextTransparent">#216C2E</color>
```

Comment:
A couple of these are the same as the colorDataSubmissionConfirmationGreen and colorJobCardCollectButton. I wonder if it would make sense to define a new color with a semantic name like colorSecondary or colorPrimaryLight that could be used for all of those colors. That we if we update it in the future all the corresponding UI would be updated automatically.

Reply:
Done

Reply:
Now done.

Reply:
Thanks!